src/core/mac/sub_mac.cpp

```diff
@@ -352,7 +352,11 @@ void SubMac::HandleReceiveDone(RxFrame *aFrame, Error aError)
         // Assuming the risk of the parent missing the Enh-ACK in favor of smaller CSL receive window
         if ((mCslPeriod > 0) && aFrame->mInfo.mRxInfo.mAckedWithSecEnhAck)
         {
+#if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_LOCAL_TIME_SYNC
+            mCslLastSync = ot::TimerMicro::GetNow();
```

Comment:
I think we can remove `ot::` and use `TimerMicor::GetNow()` directly?
(since we are already in `ot` namespace).